src/components/elements/MapPolygonPanel/MapPolygonCheckPanelItem copy.tsx

```diff
@@ -0,0 +1,209 @@
+import classNames from "classnames";
+import { DetailedHTMLProps, Dispatch, HTMLAttributes, SetStateAction, useState } from "react";
+import { When } from "react-if";
+
+import Text from "@/components/elements/Text/Text";
+import Icon, { IconNames } from "@/components/extensive/Icon/Icon";
+import ModalConfirm from "@/components/extensive/Modal/ModalConfirm";
+import ModalWithLogo from "@/components/extensive/Modal/ModalWithLogo";
+import ModalWithMap from "@/components/extensive/Modal/ModalWithMap";
+import { useModalContext } from "@/context/modal.provider";
+
+import Button from "../Button/Button";
+import Menu from "../Menu/Menu";
+import { MENU_PLACEMENT_RIGHT_BOTTOM } from "../Menu/MenuVariant";
+
+export interface MapPolygonCheckPanelItemProps
+  extends DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement> {
+  uuid: string;
+  title: string;
+  isSelected?: boolean;
+  refContainer?: React.RefObject<HTMLDivElement> | null;
+  setEditPolygon?: Dispatch<SetStateAction<boolean>>;
+  status: string;
+  polygon?: string[];
+}
+
+const MapPolygonCheckPanelItem = ({
+  title,
+  isSelected,
+  className,
+  refContainer,
+  setEditPolygon,
+  polygon,
+  status,
+  ...props
+}: MapPolygonCheckPanelItemProps) => {
+  const { openModal, closeModal } = useModalContext();
+  const [openCollapse, setOpenCollapse] = useState(true);
+  const openFormModalHandlerRequestPolygonSupport = () => {
+    openModal(
+      <ModalWithMap
+        title="Request Support"
+        onClose={closeModal}
+        content="Faja Lobi Project&nbsp;&nbsp;•&nbsp;&nbsp;Priceless Planet Coalition"
```

Comment:
Looks like lots of mocked data in this component, like this. Is it reasonable to get the mocked data out before we merge this PR?

Reply:
I have removed  file as it contained mocked data that was not being used. Thanks for pointing it out.